CHANGELOG.md

```diff
@@ -0,0 +1,83 @@
+# Changelog
+
+<!-- , -->
+<!-- #and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html). -->
```

Comment:
Perhaps we can just write: `This project adheres to Semantic Versioning: https://semver.org/spec/v2.0.0.html.`  ? Since it is a comment, markdown formatting is not needed here.